shared/checks/oval/installed_OS_is_rhel8_0.xml

```diff
@@ -0,0 +1,42 @@
+<def-group>
```

Comment:
Are `chared/checks/oval/*.xml` JINJA-enabled? Could these be macro-ized?

Reply:
I wondering that as well, but I haven't checked that yet.

Reply:
No worries, it is outside the scope of the PR :-) I might suggest it to @Mab879 if he wants to try his hand at another PR. It seems like a lot of the (Linux) OS-related version checks could be simplified.

Reply:
They are jinja enabled. We use this for OpenShift https://github.com/ComplianceAsCode/content/blob/master/shared/checks/oval/installed_app_is_ocp4.xml#L46